_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -12,6 +12,12 @@
 	exit;
 }
 
+// This would fatal on versions below WP 4.4
+global $wp_version;
+if ( ! function_exists( 'rest_api_init' ) || version_compare( $wp_version, '4.4-z', '<=' ) ) {
+	wp_die( __( 'Minimum WordPress version 4.4 required to unlock the power of the WP REST API', 'jetpack' ) );
+}
```

Comment:
This caused me some trouble when testing on older WordPress versions. As soon as Jetpack detects an old WP installation, it kills the admin, and you can't even deactivate the plugin via WP-CLI. What do you think we implement a mechanism that will deactivate Jetpack when it detects a version mismatch?